src/main/Config.cpp

```diff
@@ -275,10 +276,24 @@ readInt(ConfigItem const& item, T min = std::numeric_limits<T>::min(),
         throw std::invalid_argument(fmt::format("invalid '{}'", item.first));
     }
     int64_t v = item.second->as<int64_t>()->get();
-    if (v < min || v > max)
+    if (std::is_signed_v<T>)
```

Comment:
C++ 17 makes this kind of things so nice

Reply:
Quite!